api-reference/assets/vehicles/manufacturers/get-all.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Get Manufacturers'
+openapi: 'GET /v3/assets/vehicles/manufacturers'
+---
+
+#### Sort Support
```

Comment:
I like this, I'll go back to some other endpoints and add it. I wish there was a nicer way of automating it